package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "reservation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reservly/reservation.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/reservly/reservation/issues"
+  },
+  "homepage": "https://github.com/reservly/reservation#readme",
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-plugin-import": "^2.18.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.2",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-preset-react": "^6.24.1",
+    "jest": "^24.8.0"
+  },
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "webpack": "^4.35.2"
```

Comment:
@poema-cavalcanti webpack should not be a dependency for the application; if you want to list it, then the tools that it needs to run in your webpack.config.js file should be listed as dependencies as well (e.g., loaders)

Reply:
I forgot to remove those from package.json when first incorporating similar feedback to not include dependencies that will be used, but aren't yet. XD